### PR TITLE
Adding information about Rotoye Batmon, editing batt_smbus driver docs

### PR DESCRIPTION
**Pull request by nbelanger99**
This doc update corresponds with a pending [PR](https://github.com/PX4/PX4-Autopilot/pull/16723) for Firmware, adding support for Rotoye Batmon devices and abstracting a generic smbus smart-battery driver

### CODE REVIEW

File: en/modules/modules_driver.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Source: [drivers/batt_smbus](https://github.com/PX4/Firmware/tree/master/src/dri
```
Comment (hamishwillee) [inline en/modules/modules_driver.md:84]: @nbelanger99 Please undo the changes in this document. This is auto generated from the in-source comments in PX4-Autopilot. You can propose changes in the source separately.

Comment (nbelanger99) [reply en/modules/modules_driver.md:84]: Ah, I see, thank you. This is not true of the other files changed in this PR, correct?

File: en/smart_batteries/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Smart Batteries
+
+Smart Batteries keep track of the state of the battery, including its exact state of charge, cell voltages, time-to-empty, and temperature readings. This allows for easier flight planning and more accurate prediction of failure conditions: [Safety > Low Battery Failsafe](../config/safety.md#low-battery-failsafe).
+
+
+
+:::note
```
Comment (hamishwillee) [inline en/smart_batteries/README.md:7]: This is the driver for a battery connected to the SMBUS right (i.e. I2C)? That's just one way to connect a battery - for example, we often recommend MAVLink.

Can you please change this from a note to a new section at the very end

---

## Further Information

- [batt_smbus](../modules/modules_driver.md) - PX4 SMBus Battery Driver docs
- [MAVLink Battery Protocol](https://mavlink.io/en/services/battery.html)

File: en/smart_batteries/rotoye_batmon.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<br/><br/>
+<img src="../../assets/hardware/smart_batteries/rotoye_batmon/smart-battery-rotoye.jpg" alt="Board" width="360"/>
+
+
+## Purchase
+
+[Rotoye Store](https://rotoye.com/batmon/): Batmon kits, custom smart-batteries, and accessories
+
+
+## Wiring/Connections
```
Comment (hamishwillee) [inline en/smart_batteries/rotoye_batmon.md:15]: 1. I think it is worth here copying in the PX4 setup and https://github.com/rotoye/batmon_reader#px4
2. Also ideally the additional images
   image
   Or even better an end to end showing battery, batmon, pixhawk 4 ...
3. If you don't want to add the info about the cell to batmon wiring, add a note saying - "Information on how to connect batteries to the battery monitor is covered here.

**Comment by hamishwillee** [comment]
Looks good. I've added some notes. FYI, can you shrink **smart-battery-rotoye-xt90.png** ? It is over a MB which is big for a slow link - and I don't think needed for this diagram.

File: en/modules/modules_driver.md (hunk 1 above)
Comment (hamishwillee) [reply en/modules/modules_driver.md:84]: Correct, this is the only one in source code.

File: en/smart_batteries/rotoye_batmon.md (hunk 3 above)
Comment (nbelanger99) [reply en/smart_batteries/rotoye_batmon.md:15]: I went ahead and made those changes; I used the above image rather than the large one of the XT90.

Comment (nbelanger99) [reply en/smart_batteries/rotoye_batmon.md:15]: As for the cell connection, new versions of Batmon now connect directly to the standard 2.54mm balance leads on LiPo/LiIo batteries; that diagram on Github is out of date

Comment (hamishwillee) [reply en/smart_batteries/rotoye_batmon.md:15]: If you can show an updated version of the battery connection that would be really cool, though not in any way mandatory.

**Comment by hamishwillee** [comment]
Thanks. I merged and did a few modifications in https://github.com/PX4/PX4-user_guide/pull/1044

There is a question there for you. It will require update once the firmware PR goes in to remove the build instructions.
